README.md

```diff
@@ -0,0 +1,67 @@
+# building the current images
+
+**step 0: clone the repos**
+
+`git clone https://github.com/geoHeil/geodocker-geomesa.git`
+
+**step 1: choose correct settings**
+
+- choose settings for accumulo / geomesa / ... version which fits your environment
+- if there are major changes you might need to recompile geomsa as well
+	```
+	git clone https://github.com/locationtech/geomesa.git
+	cd geomesa
+	git checkout geomesa_2.11-1.3.0
+	update ./build.sh to desired version numbers then
+	build/mvn clean install -DskipTests=true #test failures otherwise (timeouts on macbook)
```

Comment:
GeoMesa releases are available on LocationTech's Nexus repo.  Would it be possible to skip this step?